### DISCUSSION TITLE
Never can find the release notes

### DISCUSSION THREAD

**Discussion by dknia** [UtilitechAS/amsreader-firmware#854] [Q&A] [answered]
Hi.

I love the AMS, and the improvements over time. Customer service is outstanding!

I have one frustration, that I'm sure is entirely my own fault.

Whenever there are upgrades, I just find it very hard to find the release notes. And sometimes - like today - I fail completely.

Is it possible to have a Discussion category dedicated for release notes? And maybe even a link to them from the front page of the project. It would be such a big help for me, and others equally challenged to find the release notes.

PS: Still would like a hint to where they are currently ;-)

TIA

**Comment by ArnieO** [comment, marked as answer]
Hi TIA, 

Thank you for your kind words.
The release notes are in fact easy to find - if you just know where to look!

1. Go to the main page of the Github repository: https://github.com/UtilitechAS/amsreader-firmware
2. In the rightmost column, there is a "Releases" section: 
    ![image](https://github.com/user-attachments/assets/7c43592a-660b-4efe-9ad8-97ab1e8de322)
3. Click on the heading "Releases": 
    ![image](https://github.com/user-attachments/assets/83ee96c1-85b5-4b00-9d2b-4fd20091c111)
4. You will now have a full view of the latest releases, including release notes.

If you only want to see the details regarding latest release, click on it on the main page (in stead of the heading):
![image](https://github.com/user-attachments/assets/298d2199-5aa5-4623-8386-6c235e2757a9)

**Reply by dknia** [reply]
OMG - I'm a blind idiot- sorry for that! And thanks for your patient response :-)
